apps/core/scenes/resultDetail/resultDetailContent/ResultDetailContent.js

```diff
@@ -59,12 +92,17 @@ class ResultDetailContent extends React.Component<Props, State> {
       this.props.itinerary?.price?.amount,
       this.props.itinerary?.price?.currency,
     );
+
     return (
       <>
         <ContentContainer>
           <ItinerarySectorDetails itinerary={this.props.itinerary} />
           <ResultDetailPassenger itinerary={this.props.itinerary} />
           <ContactDetailsForm
+            disableEmail={Boolean(this.props.userId)}
```

Comment:
Looks good. Maybe just one small thing to consider. We can probably allow user to set different contact email for booking, just prefill their email from login as default value. But if it will create some complications we can even keep it as it is right now.

Reply:
I was also thinking about that but in the current state of the application, it is nice to have, and it will better do it in separate PR. So I will create an issue for that.

Reply:
Ok, so lets merge this one 🚀